### ISSUE TITLE
[demo] Remove the old secops demo and the docker-compose demo profile.

### ISSUE THREAD

**Issue by ryzhyk** [feldera/feldera#2660]
Once #2654 is merged, we should be able to get rid of the old secops demo. This will simplify the onboarding experience, so I think it's worth doing asap:

* Remove the demo contaner from dockerfile.
* Remove the `demo` profile from docker compose
* Only the `manager` container should run by default.
* Document somewhere how to start Redpanda if the user wants to test Feldera with Kafka input/output.
* Remove references to the secops demo from docs:
  * docs/docker.md
  * README.md
  * ???
* This should probably be a separate issue but this [page](https://docs.feldera.com/tour/) is very outdated already. We should probably disable it for now, and re-work it in the future

**Comment by lalithsuresh** [comment]
I'd like it if we could generate example pages in the docs from the SQL files.

**Comment by snkas** [comment]
Sounds good, let's tackle this step-wise :+1: This sounds like a good start

**Comment by ryzhyk** [comment]
> It would be great if we could generate example pages in the docs from the SQL files. We already have the markdown-like comments etc.

Yep, that'd be amazing. Separate issue though.
